core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -1,1046 +0,0 @@
-import org.apache.kafka.common.message.AlterUserScramCredentialsResponseData.AlterUserScramCredentialsResult
-import org.apache.kafka.common.message.CreatePartitionsRequestData.CreatePartitionsTopic
-import org.apache.kafka.common.message.CreateTopicsRequestData.CreatableTopic
-import org.apache.kafka.common.message.CreateTopicsResponseData.{CreatableTopicConfigs, CreatableTopicResult}
-import org.apache.kafka.common.message.{AlterUserScramCredentialsRequestData, AlterUserScramCredentialsResponseData, DescribeUserScramCredentialsResponseData}
-import org.apache.kafka.common.message.DescribeUserScramCredentialsResponseData.CredentialInfo
-import org.apache.kafka.common.metrics.Metrics
-import org.apache.kafka.common.security.scram.internals.{ScramMechanism => InternalScramMechanism}
-import org.apache.kafka.server.policy.{AlterConfigPolicy, CreateTopicPolicy}
-import org.apache.kafka.server.policy.CreateTopicPolicy.RequestMetadata
-import org.apache.kafka.common.protocol.Errors
-import org.apache.kafka.common.quota.{ClientQuotaAlteration, ClientQuotaEntity, ClientQuotaFilter, ClientQuotaFilterComponent}
-import org.apache.kafka.common.requests.CreateTopicsRequest._
-import org.apache.kafka.common.requests.{AlterConfigsRequest, ApiError}
-import org.apache.kafka.common.security.scram.internals.{ScramCredentialUtils, ScramFormatter}
-import org.apache.kafka.common.utils.{Sanitizer, Utils}
-import org.apache.kafka.server.common.AdminOperationException
-import org.apache.kafka.server.config.{ConfigType, QuotaConfig, ZooKeeperInternals}
-import org.apache.kafka.server.config.ServerLogConfigs.CREATE_TOPIC_POLICY_CLASS_NAME_CONFIG
-import org.apache.kafka.server.config.ServerLogConfigs.ALTER_CONFIG_POLICY_CLASS_NAME_CONFIG
-import org.apache.kafka.server.purgatory.{DelayedOperation, DelayedOperationPurgatory}
-import org.apache.kafka.storage.internals.log.LogConfig
-
-import scala.collection.{Map, mutable, _}
-import scala.jdk.CollectionConverters._
-
-object ZkAdminManager {
-  def clientQuotaPropsToDoubleMap(props: Map[String, String]): Map[String, Double] = {
-    props.map { case (key, value) =>
-      val doubleValue = try value.toDouble catch {
-        case _: NumberFormatException =>
-          throw new IllegalStateException(s"Unexpected client quota configuration value: $key -> $value")
-      }
-      key -> doubleValue
-    }
-  }
-}
-
-
-class ZkAdminManager(val config: KafkaConfig,
-                     val metrics: Metrics,
-                     val metadataCache: MetadataCache,
-                     val zkClient: KafkaZkClient) extends Logging {
-
-  this.logIdent = "[Admin Manager on Broker " + config.brokerId + "]: "
-
-  private val topicPurgatory = new DelayedOperationPurgatory[DelayedOperation]("topic", config.brokerId)
```

Comment:
@m1a2st this change removes the metrics so could you please add it to `zk2kraft.html`?

Reply:
Sure, I will open a PR to addresss it.